Eventually becoming a full pass (start at the top)

mih opened the pull request.

mih committed:
Some affiliations are missing

mih committed:
Reduce redundancy in summary

mih committed:
Add some more graspable information to the summary

mih committed:
Shorten sentences and structure statement of need

Remove explicit git/annex mention, next section is dedicated to that.

mih committed:
Pass over Git + annex

Subjectively clearer description of git-annex does.

yarikoptic commented on paper.md:
```diff
@@ -24,6 +24,7 @@ authors:
    orcid: 0000-0003-2917-3450
    affiliation: 2
  - name: Jason Gors  # 60 commits, issues: opened 2 participated in 1
+   affiliation: FIXME
```
Is having affiliation strictly required? Would be very non inclusive from joss to forbid citizen scientists from publishing

mih replied:
That is not my concern. My concern is that without an affiliation declaration they all get `1`, which is Dartmouth College ATM. If that is OK, or even intentional, I suggest to make that explicit, rather than rely on that implicit behavior.

Screenshot without FIXME:

![image](https://user-images.githubusercontent.com/136479/115115027-744a4c80-9f92-11eb-993e-e9c6d7fe6a1c.png)

yarikoptic replied:
ah, nicely spotted -- I have missed that and it was not intentional.

mih committed:
Questions be questions

Bring DataLad more into the focus in the second one

mih committed:
Minor rephrasing of doc section

But see https://github.com/datalad/datalad-paper-joss/issues/66

mih committed:
Reducing fragility of handbook links

Remove resolved redirects, use r.html where possible, remove deep link
to use case section (other use cases are elsewhere in the handbook).

mih committed:
Reword installation section

Abuse to namedrop HPC, add convenience links that should be fairly
stable.

yarikoptic commented on paper.md:
```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
+
+[^1]: https://en.wikipedia.org/wiki/Git#Adoption
+
+## Why does DataLad add to Git and git-annex?
```
odd if not incorrectly formed sentence breaking alignment with paragraph titles ATM (but might be appropriate for #64 ). I will for now revert to the original

```suggestion
## Why Git and git-annex alone are not enough?
```

yarikoptic commented on paper.md:
```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
+
+## Why Git and git-annex?
+
+Git is the standard version control system for software development[^1].
```
there is no standard for designing version control systems, and "**the** standard" really throws me off as hinting on it to be the only one. What about simpler

```suggestion
Git is the most popular version control system for software development[^1].
```

yarikoptic commented on paper.md:
```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
+At the same time, Git is not designed to efficiently handle large (e.g., over a gigabyte) or binary files [see, e.g., @opensource:git-binary].
+This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files, or even to fully remove file content without severely impacting the integrity of a repository.
+Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.
+File content managed by git-annex is placed into a managed repository annex, instead of committing it directly
+to Git.
```
Just for consistency in formatting
```suggestion
File content managed by git-annex is placed into a managed repository annex, instead of committing it directly to Git.
```

yarikoptic commented on paper.md:
```diff
@@ -151,40 +151,43 @@ bibliography: paper.bib
+Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.
+File content managed by git-annex is placed into a managed repository annex, instead of committing it directly
+to Git.
+Instead of the file content, git-annex only commits a compact reference that enables identification and association of file name and content.
```
```suggestion
Instead of the file content, git-annex only commits a compact reference that enables identification and association of a file name with the content.
```
since it is not necessarily a reversible relationship, and content might be associated with multiple files

yarikoptic reviewed (commented):
Reads great, thanks! I have suggested a few minor corrections, which I would merge (e.g. in an hour or two) along with the PR if no objections are expressed.

yarikoptic committed:
Minor tune ups to @mih's changes
